server/src/main/kotlin/lib/koin/KoinInstance.kt

```diff
@@ -0,0 +1,9 @@
+package lib.koin
+
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+inline fun <reified T> getKoinInstance(): T =
+    object : KoinComponent {
+        val value: T by inject()
+    }.value
```

Comment:
Something like this should work.
```suggestion
inline fun <reified T> getKoinInstance(): T =
    KoinPlatformTools.defaultContext().get().get<T>()
```